src/components/Modal/Modal.css

```diff
@@ -49,6 +61,8 @@
 }
 
 .post-submit-btn:hover {
+  cursor: pointer;
```

Comment:
This could be DRYed up with the other button: `.post-submit-btn:hover, .close-modal:hover {stuff}` or potentially just do all buttons:hover

Reply:
cursor styling has been dried up.